app/views/errors/internal_server_error.html.erb

```diff
@@ -1,8 +1,9 @@
 <main role="main" class="govuk-main-wrapper" id="main-content">
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-xl">We're sorry, but something went wrong.</h1>
-      <p class="govuk-body">If you are the application owner check the logs for more information.</p>
+      <h1 class="govuk-heading-xl">Sorry, there is a problem with the service</h1>
+      <p class="govuk-body">We could not proceed with calculating your duty.</p>
+      <p class="govuk-body">Click here to <%= link_to('start again', import_date_path) %>.</p>
```

Comment:
Should we pass the query params to the import_date_path if they're available?

Reply:
Yeah, interesting point. They would already have it on the session though?